### DISCUSSION TITLE
Stale Issues

### DISCUSSION THREAD

**Discussion by SkyeHoefling** [dnnsoftware/Dnn.Platform#4332] [General] [answered]
I've noticed several long-term initiatives are being marked as stale. Could we add a special label that the stale-bot will ignore. This will make it easier for maintainers and contributors from having to comment just to keep eyes on these items.

Some items that come to mind
- Logging Abstraction
- Component Factory -> Dependency Injection
- Data Provider -> Dependency Injection
- etc.

**Comment by mitchelsellers** [comment, marked as answer]
We have this with pinned

**Reply by SkyeHoefling** [reply]
Thanks! I'll ask on a couple issues to get them marked as pinned
